netneurotools/plotting.py

```diff
@@ -416,6 +416,32 @@ def plot_fsaverage(data, *, lhannot, rhannot, order='lr', mask=None,
     -------
     brain : surfer.Brain
         Plotted PySurfer brain
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from netneurotools.datasets import fetch_cammoun2012, \
+                                           fetch_schaefer2018
+    >>> from netneurotools.plotting import plot_fsaverage
+
+    Plotting with the Cammoun 2012 parcellation we specify `order='RL'` because
+    many of the Lausanne connectomes have data for the right hemisphere before
+    the left hemisphere.
+
+    >>> values = np.random.rand(219)
+    >>> cammoun = fetch_cammoun2012('fsaverage')['scale125']
+    >>> plot_fsaverage(values, order='RL',
+    ...                lhannot=cammoun.lh, rhannot=cammoun.rh) # DOCTEST: +SKIP
```

Comment:
Okay, totally my bad here. This should be

```suggestion
    ...                lhannot=cammoun.lh, rhannot=cammoun.rh)  # doctest: +SKIP
```